interface/forms/LBF/printable.php

```diff
@@ -99,6 +104,18 @@
         'keep_table_proportions' => true
     );
     $pdf = new mPDF($config_mpdf);
+    $pdf->SetHTMLHeader('
+		<div style="text-align: right; font-weight: bold;">
+			'.$patientname.' DOB: '.$patientdob.'
+		</div>');
+    $pdf->SetHTMLFooter('
+		<table width="100%">
```

Comment:
Why a `table` over a `div`?

Reply:
In doing something quick and dirty... I followed the example given
https://mpdf.github.io/headers-footers/method-2.html

It was a copy-paste. No more thought in it than that. When you're under the gun and need to turn something out quick. This is what happens.

Reply:
Yep, I get it. The academics of how something should be programmed will always lose out to the necessitates of business demand.